spec/support/capybara.rb

```diff
@@ -2,17 +2,22 @@
 
 require 'capybara/rails'
 require 'capybara/rspec'
-require 'capybara-webkit'
 require 'capybara-screenshot/rspec'
+require 'selenium/webdriver'
 
-Capybara.javascript_driver = :webkit
+Capybara.register_driver :chrome do |app|
+  Capybara::Selenium::Driver.new(app, browser: :chrome)
+end
 
-Capybara::Webkit.configure do |config|
-  config.block_unknown_urls
-  config.timeout = 20
-  config.ignore_ssl_errors
-  # NOTE: Do we need to raise them?
-  # config.raise_javascript_errors = true
+Capybara.register_driver :headless_chrome do |app|
+  capabilities = Selenium::WebDriver::Remote::Capabilities.chrome(
+    # NOTE: Might need to include `disable-gpu` param
```

Comment:
we may need `--no-sandbox` as well

Reply:
Good point. Codeship runs all builds in containers, so `sandbox` might be useful.